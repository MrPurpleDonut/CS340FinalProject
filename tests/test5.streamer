text contents << "/path/to/file";
error
{
  print "No such file";
  exit 1;
}
print contents;
