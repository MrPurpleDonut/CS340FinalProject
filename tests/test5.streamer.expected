Program parsed correctly!
---------------------------
--------Printing AST-------
---------------------------
Program
    Function List
    Statement List
        Declaration statement: contents
            Type: text
            File Read into variable: contents
                Quoted string: "/path/to/file"
        Error Statement
            Statement List
                Print Statement
                    Quoted string: "No such file"
                Exit Statement
                    Number: 1.0
        Print Statement
            Identifier: contents
